mlir/docs/Bufferization.md

```diff
@@ -165,7 +165,7 @@ such as a subsequent read of `%s`).
 
 RaW conflicts are detected with an analysis of SSA use-def chains (details
 later). One-Shot Bufferize works best if there is a single SSA use-def chain,
-where the result of a tensor op is the "destination" operand of the next tensor
```

Comment:
The quote here are meant to express that this isn't really a "destination" at the SSA level but only an input for the operation.